### PR TITLE
Expose AVATAR_URL_PREFIX for docker deployment

### PR DESCRIPTION
**Pull request by dankrzeminski32**
## Proposed changes

Fixes [#8210](https://github.com/WeblateOrg/weblate/issues/8210)

Allows AVATAR_URL_PREFIX to be set via environment variable.

## Checklist

- [x] Lint and unit tests pass locally with my changes.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [x] I have added documentation to describe my feature.
- [ ] I have squashed my commits into logic units.
- [x] I have described the changes in the commit messages.

### CODE REVIEW

File: weblate/settings_docker.py [hunk 1]
```diff
@@ -1301,6 +1301,7 @@
 
 # Avatars
 ENABLE_AVATARS = get_env_bool("WEBLATE_ENABLE_AVATARS", True)
+AVATAR_URL_PREFIX = os.environ.get("WEBLATE_AVATAR_URL_PREFIX", "")
```
Comment (nijel) [inline weblate/settings_docker.py:1304]: It should keep the default value:

```suggestion
AVATAR_URL_PREFIX = os.environ.get("WEBLATE_AVATAR_URL_PREFIX", "https://www.gravatar.com/")
```

Comment (dankrzeminski32) [reply weblate/settings_docker.py]: Understood, I pushed those changes, the correct default value is now used.

File: docs/admin/install/docker.rst [hunk 2]
```diff
@@ -840,6 +840,12 @@ Generic settings
 
    Configures :setting:`ENABLE_AVATARS`.
 
+.. envvar:: WEBLATE_AVATAR_URL_PREFIX
+
+   .. versionadded:: 4.14.3
```
Comment (nijel) [inline docs/admin/install/docker.rst:845]:
```suggestion
   .. versionadded:: 4.15
```

**Comment by nijel** [comment]
Merged, thanks for your contribution!

**Pull request merged**